tools/rdiff-backup.spec.template-fedora

```diff
@@ -63,7 +63,7 @@ rm -rf $RPM_BUILD_ROOT
 %{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/*.so
 %{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup-*-py?.?.egg-info
 %ghost %{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/__pycache__/*.pyo
-%doc CHANGELOG COPYING FAQ.html examples.html README.md DEVELOP.md
+%doc CHANGELOG COPYING FAQ.md examples.md README.md DEVELOP.md
```

Comment:
I assume that it should read with `docs/FAQ.md` etc...

Reply:
Similar to above.